pkg/experiment/sensitivity/factory.go

```diff
@@ -121,6 +125,12 @@ type ExecutorFactoryFunc func(isolation.Decorators) (executor.Executor, error)
 
 // Create returns aggressor of chosen type using provided executorFactor to create executor.
 func (f AggressorFactory) Create(name string, executorFactory ExecutorFactoryFunc) (executor.Launcher, error) {
+
+	// Zero-value sensitivity.LauncherSessionPair represents baselining.
+	if name == NoneAggressorID {
+		return nil, nil
```

Comment:
Sometime in future it would be nice to return some dummy Launcher with dummy TaskHandle, so that we won't be returning nils.